pkgs/apps/iboview/default.nix

```
@@ -0,0 +1,48 @@
+{ lib, stdenv, wrapQtAppsHook, qmake, qtscript, qtsvg, blas-ilp64, lapack-ilp64
+, glew, boost, eigen, libGLU }:
+
+stdenv.mkDerivation rec {
+  pname = "IboView";
+  version = "20211019";
+
+  src = builtins.fetchurl {
```

```suggestion
  src = fetchurl {
```

That should be the `fetchurl` from nixpkgs or I get:
```
error: access to URI 'http://www.iboview.org/bin/ibo-view.20211019.tar.bz2' is forbidden in restricted mode
```